silk/tools/otns_manager.py

```diff
@@ -1056,8 +1056,12 @@ def unsubscribe_from_all_nodes(self):
       subscriber.unsubscribe()
     self.otns_monitor_map.clear()
 
-  def update_layout(self):
+  def update_layout(self, use_two_layer=False):
     """Update layout of nodes in auto layout mode.
+
+    Args:
+      use_two_layer (bool, optional): if layout uses separate circles for
```

Comment:
Question: can we use two layers but still stationary?

Reply:
What does stationary mean in that case? Currently if two layers is used, both circles have the number of spots equal to total number of nodes, and the nodes will move to inner or outer circle depending on their role, but occupying the spots with the same indices so that movement is minimized.

Reply:
I mean we can keep the nodes in two layers, but somehow make them move less than often. 
For example, if a Router becomes Child, we delay the movement for a while until the found it to be a stable Router, then we move it to inner layer. Same can be done for a node to downgrade from Router to Child.
This can make the topology visualization more stable?

BTW, we can move the node using a smooth animation instead of sudden position change to have a better UX. 

Thoughts?

Reply:
It does sound like a good idea. I think it might be a bit complex to implement the delay, and the main issue I was seeing wasn't nodes shifting often between two circles, but that circles being asymmetrical looks a bit less nice for many clusters.

Animated move sounds like a good idea. Needs a change on OTNS side?

Reply:
Yes, animation can (and maybe better) be implemented on OTNS side.